src/utils/dates.js

```diff
@@ -1,6 +1,7 @@
 /* eslint no-fallthrough: off */
 import dateMath from 'date-arithmetic'
 import localizer from '../localizer'
+import moment from 'moment'
```

Comment:
can't use moment, it's not a direct dependency. use the functions from dateMath, instead (it's got the same api)